deployer/commands/validate/config.py

```diff
@@ -236,6 +235,73 @@ def authenticator_config(
             )
 
 
+@validate_app.command()
+def configurator_config(
+    cluster_name: str = typer.Argument(..., help="Name of cluster to operate on"),
+    hub_name: str = typer.Argument(None, help="Name of hub to operate on"),
+):
+    """
+    For each hub of a specific cluster:
+     - It asserts that when the singleuser configuration overrides the same fields like the configurator,
```

Comment:
```suggestion
     - It asserts that when the singleuser configuration overrides the same fields like the configurator, specifically kubespawner_override and profile_options,
```

I think we could make this a little bit more explicit, especially if someone reading this doesn't know what the configurator does

Reply:
Yes, very good point! Thank you! ✨